rites/roman1969/src/locales/pt-br.ts

```diff
@@ -465,6 +480,7 @@ export const locale: Locale = {
     peter_claver_priest: 'São Pedro Claver, presbítero',
     peter_damian_bishop: 'São Pedro Damião, bispo e doutor da Igreja',
     peter_julian_eymard_priest: 'São Pedro Juliano Eymard, presbítero',
+    peter_of_alcantara_priest: 'São Pedro de Alcântara, presbítero', // src: Calendário Próprio do Brasil - CNBB
```

Comment:
as above w/ the source

Reply:
i can't seem to find his date reference at all on the CNBB website. i'm pushing the fixes for sources & notes i can find, but this one will need to be fixed before we can merge